packages/webapp/components/Auction/GnarInfo.tsx

```diff
@@ -74,15 +55,15 @@ export const GnarInfo: FC<GnarInfoProps> = ({ isOg, gnarData, ...props }: GnarIn
   const isBurned = !!gnarData.gnar.auction && auctionEnded && !winner
   const isClaimedGnar = !isTreasuryGnar && !gnarData.gnar.auction
   return (
-    <VStack w={"full"} alignItems={"start"} spacing={6} maxW={{ base: "full", lg: "500px", xl: "xl" }} {...props}>
+    <VStack w={"full"} spacing={6} maxW={{ base: "full", lg: "500px", xl: "xl" }} {...props}>
```

Comment:
For whatever reason, after trying multiple different approaches, it was this one prop removal that allowed the "Under Construction" headings to automatically center on mobile-sized screens